build-scripts/gulp/download-translations.js

```diff
@@ -89,7 +92,73 @@ gulp.task("check-all-files-exist", async function () {
   await Promise.allSettled(writings);
 });
 
+const lokaliseProjects = {
+  backend: "130246255a974bd3b5e8a1.51616605",
+  frontend: "3420425759f6d6d241f598.13594006",
+};
+
+gulp.task("fetch-lokalise", async function () {
+  let apiKey;
+  try {
+    apiKey = await fs.readFile(".lokalise_token");
```

Comment:
To return a string and not a buffer:

```suggestion
    apiKey = await fs.readFile(".lokalise_token", "utf-8");
```